transaction/src/view.rs

```diff
@@ -1,26 +1,88 @@
+use anyhow::Context;
+use bytes::Bytes;
 use decaf377_fmd::Clue;
-use penumbra_crypto::{memo::MemoPlaintext, transaction::Fee, AddressView};
+use decaf377_rdsa::{Binding, Signature};
+use penumbra_crypto::{
+    memo::{MemoCiphertext, MemoPlaintext},
+    transaction::Fee,
+    AddressView,
+};
 use penumbra_proto::{core::transaction::v1alpha1 as pbt, DomainType};
+
 use serde::{Deserialize, Serialize};
 
 pub mod action_view;
 mod transaction_perspective;
 
 pub use action_view::ActionView;
+use penumbra_tct as tct;
 pub use transaction_perspective::TransactionPerspective;
 
+use crate::{Action, Transaction, TransactionBody};
+
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(try_from = "pbt::TransactionView", into = "pbt::TransactionView")]
 pub struct TransactionView {
+    pub body_view: TransactionBodyView,
+    pub binding_sig: Signature<Binding>,
+    pub anchor: tct::Root,
+}
+#[derive(Clone, Debug, Default)]
+pub struct TransactionBodyView {
```

Comment:
I think we'll want this to be
```
#[derive(Clone, Debug, Serialize, Deserialize)]
#[serde(try_from = "pbt::TransactionBodyView", into = "pbt::BodyTransactionView")]
pub struct TransactionBodyView {
```
so that it's serializable